docker/docker-compose.yml

```
@@ -79,6 +79,7 @@ services:
       - mongodb:/data/db
     ports:
       - "27017:27017"
+    restart: unless-stopped
```

yamllint (1.35.1)

[error] duplication of key "restart" in mapping

(key-duplicates)